.github/workflows/ci-goreleaser.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Setup QEMU
         uses: docker/setup-qemu-action@v2
         with:
-          platforms: arm64
+          platforms: arm64,ppc64le
```

Comment:
Can you also add in the readme, or in another place you find suitable, what needs to be done to add new platforms? This problem has happened already in the past with other platforms, and I think we could have prevented it if we had clear instructions on how to add new platforms.